app/include/zmk/backlight.h

```diff
@@ -6,14 +6,10 @@
 
 #pragma once
 
-int zmk_backlight_set_on(bool on);
-bool zmk_backlight_is_on();
-
-int zmk_backlight_set_brt(int brt);
-int zmk_backlight_get_brt();
```

Comment:
I think `zmk_backlight_get_brt` was useful. Why did you remove it?

Reply:
`calc_brt` with direction 0 gives the same result thus its not needed

Reply:
See https://github.com/bortoz/zmk/blob/backlight/app/src/backlight.c#L145.
They are not the same function, `zmk_backlight_get_brt` should return 0 if `state.on = false`. It is still possible to calculate this value but I think it is convenient to have this function.

Reply:
Fair enough, I will implement it